doc/devel/documenting_mpl.rst

```diff
@@ -19,6 +24,31 @@ The requirements are as follows (https://github.com/matplotlib/matplotlib/blob/m
 6. pillow
 7. graphviz
 
+.. note::
+
+  * You'll need a minimal working latex distribution for many examples to run.
+  * `Graphviz <http://graphviz.readthedocs.io/en/latest/manual.html>`_ is not pip-installable so you need
```

Comment:
This link is not to the right graphviz. It is [here](http://www.graphviz.org/Download.php).